src/cast_core/include/cast_core/actors/OptionsConversion.hpp

```diff
@@ -222,6 +222,32 @@ struct NodeConvert<mongocxx::options::transaction> {
         return rhs;
     }
 };
-}  // namespace YAML
 
+template <>
+struct NodeConvert<mongocxx::options::update> {
+    using type = mongocxx::options::update;
+
+    static type convert(const Node& node) {
+        type rhs{};
+
+        if (node["BypassDocumentValidation"]) {
+            rhs.bypass_document_validation(node["BypassDocumentValidation"].to<bool>());
```

Comment:
In the future you may wish to use the if-with-initializer C++17ism:

```c++
if (const auto& bypass = node["Bypass"]; bypass) {
  rhs.bypass(bypsas.to<bool>());
}
```

Existing code doesn't use this much so keeping with existing practice is also reasonable.

Reply:
c++17 is ok to use in genny now?